### PR TITLE
[RFR][1LP] adding pod support to ext auth code

### PR DESCRIPTION
**Pull request by izapolsk**
these changes were tested against pod locally.
{{ pytest: cfme/tests/integration/test_cfme_auth.py }}

### CODE REVIEW

**Comment by izapolsk** [comment]
This PR is about  to fix some of broken authentication tests and add pod environment support.
This PR isn't intended to fix all the cfme auth tests because some of them require changes in infrastructure and have to be refactored.

File: cfme/configure/access_control.py [hunk 1]
```diff
@@ -262,6 +262,7 @@ def _retrieve_ext_auth_user_groups(self):
 
     def add_group_from_ldap_lookup(self):
         self._retrieve_ldap_user_groups()
+        sel.wait_for_ajax()
```
Comment (RonnyPfannschmidt) [inline cfme/configure/access_control.py]: **OPTIONAL:** do we have a widgettastic version of that one

Comment (izapolsk) [inline cfme/configure/access_control.py]: I'd say these tests need to be completely rewritten and according to dajo mshriver is going to do that soon
So, I added as easy fix as it is possible

Comment (mshriver) [inline cfme/configure/access_control.py]: I think mpusater is actually going to end up being the 'author' and I'll be ghost writing so to speak, but yes the ext auth tests need re-written.

**Pull request merged**